aws_neuron/assets/dashboards/aws_neuron_overview.json

```diff
@@ -0,0 +1,1189 @@
+              "content": "Monitoring execution status helps identify the operational state of machine learning models, ensuring they are running as expected. Tracking execution latency provides insights into the performance and efficiency of model inference, helping optimize response times and resource usage.",
+              "background_color": "green",
+              "font_size": "14",
+              "text_align": "center",
+              "vertical_align": "top",
+              "show_tick": false,
+              "tick_pos": "50%",
+              "tick_edge": "left",
+              "has_padding": true
+            },
+            "layout": {
+              "x": 0,
+              "y": 0,
+              "width": 12,
+              "height": 1
+            }
+          },
+          {
+            "id": 1102497452686722,
+            "definition": {
+              "title": "Execution status rates",
+              "title_size": "16",
+              "title_align": "left",
+              "show_legend": true,
+              "legend_layout": "auto",
+              "legend_columns": [
+                "avg",
+                "min",
+                "max",
+                "value",
+                "sum"
+              ],
+              "type": "timeseries",
+              "requests": [
+                {
+                  "formulas": [
+                    {
+                      "alias": "Executions success rate",
+                      "style": {
+                        "palette": "green"
+                      },
+                      "formula": "query1"
+                    },
+                    {
+                      "alias": "Execution error rate",
+                      "style": {
+                        "palette": "red"
+                      },
+                      "formula": "query2"
+                    }
+                  ],
+                  "queries": [
+                    {
+                      "data_source": "metrics",
+                      "name": "query1",
+                      "query": "avg:aws_neuron.execution.status.count{status_type:completed , $instance_id, $instance_type, $availability_zone, $region, $subnet_id} by {instance_id}.as_rate()"
+                    },
+                    {
+                      "data_source": "metrics",
+                      "name": "query2",
+                      "query": "avg:aws_neuron.execution.status.count{not status_type:completed and $instance_id AND $instance_type AND $availability_zone AND $region AND $subnet_id} by {instance_id}.as_rate()"
+                    }
+                  ],
+                  "response_format": "timeseries",
+                  "style": {
+                    "palette": "dog_classic",
+                    "order_by": "values",
+                    "color_order": "shuffled",
+                    "line_type": "solid",
+                    "line_width": "normal"
+                  },
+                  "display_type": "area"
```

Comment:
You keeping area here? No preference for me just wanted to ask.

Reply:
Lines may look better, I changed it